tests/fips/test_fips.py

```diff
@@ -0,0 +1,83 @@
+import logging
+import re
+
+import pytest
+
+from tests.common.helpers.assertions import pytest_assert
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.sanity_check(skip_sanity=True),
+    pytest.mark.disable_loganalyzer,  # disable automatic loganalyzer
+    pytest.mark.topology('any')
+]
+
+
+def test_fips(duthosts, enum_rand_one_per_hwsku_hostname):
+    duthost = duthosts[enum_rand_one_per_hwsku_hostname]
+
+    # check if the fips feature enabled on image level:
+    fips_status = duthost.shell("sudo sonic-installer get-fips")["stdout"]
+    if "FIPS is enabled" not in fips_status:
+        logger.info("Ignore test becauser FIPS no enabled")
```

Comment:
>Ignore test [](http://example.com/codeflow?start=21&length=11)

if ignore test, then we need to exit here?

Reply:
Thanks, fixed